template/config/webpack.base.config.js

```diff
@@ -2,9 +2,10 @@ const path = require('path');
 const project = require('./project');
 
 module.exports = {
+  mode: (process.env.NODE_ENV !== 'production') ? 'development' : 'production',
   resolve: {
     modules: ['node_modules'],
-    extensions: ['.js', '.jsx', '.json', 'less'],
+    extensions: ['.js', '.jsx', '.json', 'less', '.wasm'],
```

Comment:
`.wasm` extension is default usually, but since we specify our extensions need to add this as well!